### DISCUSSION TITLE
What happened to the LiveChart2 user guide on a server?

### DISCUSSION THREAD

**Discussion by lavrentius-dev** [Live-Charts/LiveCharts2#1712] [Q&A]
Can't seem to find the guide. Doc reads: "You can also use LiveCharts 2 on a console app or on the server side installing only the core packages, take a look at [this guide](https://lvcharts.com/docs/WPF/2.0.0-beta.330/samples.general.chartToImage#build-an-image-in-the-server-side-or-console-app)." We're past the 2.0.0-beta.330 version of course but I also can't find it on trying with 2.0.0-rc4.5 instead.

**Comment by LiamT98** [comment]
I second this. Any formal guidance on how to setup LiveCharts2 on a Blazor Web App in InteractiveServer mode? @beto-rodriguez

**Comment by beto-rodriguez** [comment]
@lcocea here is the link:

https://livecharts.dev/docs/WPF/2.0.0-rc5.4/samples.general.chartToImage#build-an-image-in-the-server-side-or-console-app

older versions (like beta.330) are no longer in the website.


@LiamT98 I am not sure if that is supported by SkiaSharp, I mean it currently works on WASM. About blazor interactive moves and according to AI:

> SkiaSharp does not natively support Blazor's interactive render modes like "InteractiveServer" or "InteractiveWebAssembly." SkiaSharp primarily focuses on graphics rendering and is often used in Blazor applications through WebAssembly (WASM) or as part of a hybrid setup with .NET MAUI. However, integrating SkiaSharp directly into Blazor's interactive render modes can be challenging due to compatibility constraints.

**Reply by LiamT98** [reply]
Thanks for the response. There's definitely a few things in play. I'm doing some more research and trying to work out why rendering the image sets server-side and streaming over SignalR for animation/interactivity wouldn't be possible. Of course latency should be considered but given a good connection I don't see an issue.

https://github.com/mono/SkiaSharp/issues/1194
This SkiaSharp issue actually details the evolution of the SkiaSharp library for use with WASM back in 2020/2021 but I get the impression that server-side use was deemed more straightforward - https://github.com/mono/SkiaSharp/issues/1194#issuecomment-644904158

I'm no expert so would be curious to know your take @beto-rodriguez
